pyramid_swagger/tween.py

```diff
@@ -131,7 +134,8 @@ def swagger_data(_):
             with validation_context(request, response=response):
                 swagger_handler.handle_response(
                     response,
-                    validator=getattr(op_or_validators_map, 'response', None))
+                    validator=getattr(op_or_validators_map, 'response', None),
+                    op=op_or_validators_map)
```

Comment:
To handle this better, could we change to interface of `handle_response()` to always just accept a single object `op_or_validators_map` ?

```python

swagger_handler.handle_response(response, op_or_validators_map)
```

That way any differences between them are hidden away behind the implementations